src/reducers/globalState.js

```diff
@@ -306,6 +306,11 @@ export default function reducer(_state, action) {
             end: state.currentRoute.duration,
             previous: state.zoom,
           }
+
+          state.loop = {
+            startTime: state.zoom.start,
+            duration: state.zoom.end - state.zoom.start,
+          };
```

Comment:
I believe this is only masking over the problem, in that `popTimelineRange` isn't used the same way when using the browser back buttons, but add an event listener seems to make the behavior even more buggy